modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordDisplayContext.java

```diff
@@ -127,6 +133,7 @@ protected DDMFormRenderingContext createDDMFormRenderingContext(
 
 		formRenderingContext.setHttpServletRequest(
 			_ddmFormAdminRequestHelper.getRequest());
+
```

Comment:
@marcosapmf, was it introduced by automatic source format? If not, please remove this blank line.